src/components/Tool/NavToolList/Item/Item.test.tsx

```diff
@@ -1,20 +1,34 @@
 import React from 'react';
+import { ThemeProvider } from '@mui/material/styles';
 import TestWrapper from '__tests__/util/TestWrapper';
 import { render } from '__tests__/util/testingLibraryReactMock';
+import theme from '../../../../theme';
 import { Item } from './Item';
 
 const item = {
   url: 'testItem',
   title: 'test title',
   isSelected: false,
+  id: 'testItem',
 };
 
 describe('ToolItem', () => {
   it('default', () => {
```

Comment:
Can we add some more tests for the different branches in the component? At least these 3 cases would be great!
* `needsAttention === false`
* `needsAttention === true && totalCount < 9`
* `needsAttention === true && totalCount > 9`